product-service/functions/getProductsList/getProductsList.js

```diff
@@ -33,8 +33,6 @@ handleResponse = (products = {}, status = 200) => ({
     body: JSON.stringify(products),
   });
 getProductsList = async (event) => {
-  console.log(12);
```

Comment:
As far as can understand you still need to log the incoming event in **every** lambda functions.